### ISSUE TITLE
Could not find shoulda-matchers-4.0.0 in any of the sources

### ISSUE THREAD

**Issue by salimane** [thoughtbot/shoulda-matchers#1184]
I am getting the above
Gemfile
```
gem 'shoulda-matchers', '3.1.3'
```
```
bundle install
.....
Could not find shoulda-matchers-4.0.0 in any of the sources
```

any idea why is this happening ?

**Comment by mcmire** [comment]
@salimane I'm not sure why that's happening. If you request 3.1.3, you should be getting 3.1.3 (I just installed it locally to make sure). I did end up yanking 4.0.0, so it's not available anymore, but that doesn't explain why you're getting this error. What happens if you say `gem install shoulda-matchers -v 3.1.3` manually?

**Comment by salimane** [comment]
@mcmire for some reasons, it started working again. I will close this. Thanks again

**Comment by mcmire** [comment]
Weird, okay!

**Comment by barmic12** [comment]
I had the same issue when I changed ruby version in my project. I sorted it out by removing Gemfile.lock and run again `bundle install`.

**Comment by victorhazbun** [comment]
is the version 4.0.0 removed? or what happened?

**Comment by victorhazbun** [comment]
I fixed it by upgrading to the version 4.1.2

`gem 'shoulda-matchers', '~> 4.1.2'`
